File: Server/src/main/java/com/api/server/foodb/Pathway.java
package com.api.server.foodb;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "pathways")
public class Pathway {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Size(max = 255)
    @Column(name = "smpdb_id")
    private String smpdbId;

    @Size(max = 255)
    @Column(name = "kegg_map_id")
    private String keggMapId;

    @Size(max = 255)
    @Column(name = "name")
    private String name;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

}

File: DotNetServer/DotNetServer/Core/Entities/Foodb/Pathway.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DotNetServer.Core.Entities.Foodb;

[Table("pathways")]
public class Pathway
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("smpdb_id")]
    [StringLength(255)]
    public string? SmpdbId { get; set; }

    [Column("kegg_map_id")]
    [StringLength(255)]
    public string? KeggMapId { get; set; }

    [Column("name")] [StringLength(255)] public string? Name { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }

    [InverseProperty("Pathway")]
    public virtual ICollection<CompoundsPathway> CompoundsPathways { get; set; } = new List<CompoundsPathway>();
}